testing/regress/ecl/genjoin.ecl

```diff
@@ -67,6 +76,9 @@ jrec xfmgrp(jrec l, DATASET(jrec) r, STRING lab) := TRANSFORM
     SELF.rstr := r[c].rstr;
     SELF.c := c;
     SELF.label := lab;
+    SELF.ml := MATCHED(l);
```

Comment:
genjoin_nothor.ecl and genjoin_justroxie.ecl have coverage of group denormalize, not sure if it's complete or not.